engine/src/flutter/impeller/renderer/backend/vulkan/descriptor_pool_vk_unittests.cc

```diff
@@ -31,29 +28,10 @@ TEST(DescriptorPoolRecyclerVKTest, ReclaimMakesDescriptorPoolAvailable) {
   {
     // Fetch a pool (which will be created).
     auto pool = DescriptorPoolVK(context);
-    pool.AllocateDescriptorSets({}, *context);
+    pool.AllocateDescriptorSets({}, 0, *context);
   }
 
-  // There is a chance that the first death rattle item below is destroyed in
-  // the same reclaim cycle as the pool allocation above. These items are placed
-  // into a std::vector and free'd, which may free in reverse order. That would
-  // imply that the death rattle and subsequent waitable event fires before the
-  // pool is reset. To work around this, we can either manually remove items
-  // from the vector or use two death rattles.
-  for (auto i = 0u; i < 2; i++) {
-    // Add something to the resource manager and have it notify us when it's
-    // destroyed. That should give us a non-flaky signal that the pool has been
-    // reclaimed as well.
-    auto waiter = fml::AutoResetWaitableEvent();
-    auto rattle = fml::ScopedCleanupClosure([&waiter]() { waiter.Signal(); });
-    {
-      UniqueResourceVKT<fml::ScopedCleanupClosure> resource(
-          context->GetResourceManager(), std::move(rattle));
-    }
-    waiter.Wait();
-  }
-
-  auto const pool = context->GetDescriptorPoolRecycler()->Get();
+  auto const pool = context->GetDescriptorPoolRecycler()->GetDescriptorPool();
```

Comment:
auto const?

Reply:
what about it?

Reply:
oh, const auto?

Reply:
> what about it?

I read this in a new york accent lol.  I don't know what `auto const` is.  It seems like it may be functionally equivalent if the deduced type is not a pointer.  Let's just remove the `auto`.